selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -245,10 +249,11 @@ void WifiUI::refresh() {
     // Clickable SSID label
     QPushButton *ssidLabel = new QPushButton(network.ssid);
     ssidLabel->setObjectName("ssidLabel");
-    ssidLabel->setEnabled(network.connected == ConnectedType::DISCONNECTED &&
-                          network.security_type != SecurityType::UNSUPPORTED);
+    ssidLabel->setEnabled(network.security_type != SecurityType::UNSUPPORTED);
```

Comment:
why enable if we're connected or connecting?

Reply:
enabled is tied to the greyed out style sheet. We only connect the button if we're disconnected below, but I can keep this the same and just set a property for the style sheet to use